src/element/list.rs

```diff
@@ -77,6 +77,12 @@ impl From<Sequence> for List {
     }
 }
 
+impl From<List> for Sequence {
+    fn from(value: List) -> Self {
+        value.0
+    }
+}
```

Comment:
🗺️ This `impl` allows a `List(Sequence)` to discard the type wrapper and become a `Sequence`. For now, it's used to convert `List(Sequence)` to `Value::List(Sequence)`.